### ISSUE TITLE
Monitor specific device (ignore others)

### ISSUE THREAD

**Issue by jcharaoui** [glensc/nagios-plugin-check_raid#183]
In the setup I'm building, I'd like to set up one specific check for each monitored raid array. The default behavior of only being able to ignore devices via the command-line, and not being able to explicitly list which device(s) to monitor) is making this more difficult than it should be.

**Comment by glensc** [comment]
indeed. while you may be able to choose which controller (plugin) to use, but not which device.

as this needs to be implemented to each plugin, which plugin you're interested in?

**Comment by jcharaoui** [comment]
I use the `mdstat` (Linux-RAID) plugin.
